### ISSUE TITLE
Some ITU-T Recommendations not fetched

### ISSUE THREAD

**Issue by ronaldtse** [relaton/relaton-itu#51]
This is from `mn-samples-itu/G.650.1-201803.adoc`:

```sh
[relaton-itu] ("ITU-T G.650.2") fetching...
[relaton-itu] WARNING: no match found online for ITU-T G.650.2. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T G.650.2 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
[relaton-itu] ("ITU-T G.651.1") fetching...
[relaton-itu] WARNING: no match found online for ITU-T G.651.1. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T G.651.1 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
[relaton-itu] ("ITU-T G.652") fetching...
[relaton-itu] WARNING: no match found online for ITU-T G.652. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T G.652 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
[relaton-itu] ("ITU-T G.653") fetching...
[relaton-itu] WARNING: no match found online for ITU-T G.653. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T G.653 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
[relaton-itu] ("ITU-T G.654") fetching...
[relaton-itu] WARNING: no match found online for ITU-T G.654. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T G.654 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
[relaton-itu] ("ITU-T G.655") fetching...
[relaton-itu] WARNING: no match found online for ITU-T G.655. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T G.655 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
[relaton-itu] ("ITU-T G.656") fetching...
[relaton-itu] WARNING: no match found online for ITU-T G.656. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T G.656 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
[relaton-itu] ("ITU-T G.657") fetching...
[relaton-itu] WARNING: no match found online for ITU-T G.657. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T G.657 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
[relaton-itu] ("ITU-T G.671") fetching...
[relaton-itu] WARNING: no match found online for ITU-T G.671. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T G.671 (all parts)".
...
[relaton-itu] ("ITU-T G.Suppl.47") fetching...
[relaton-itu] WARNING: no match found online for ITU-T G.Suppl.47. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T G.Suppl.47 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).

```

I believe all of these documents are available from ITU's site.

**Comment by ronaldtse** [comment]
And these from `mn-samples-itu/sources/T-REC-H.782-201811-I.MSW-E.adoc`:

```sh
[relaton-itu] ("ITU-T H.741.2") fetching...
[relaton-itu] WARNING: no match found online for ITU-T H.741.2. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T H.741.2 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
[relaton-itu] ("ITU-T H.780") fetching...
[relaton-itu] WARNING: no match found online for ITU-T H.780. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T H.780 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
[relaton-itu] ("ITU-T H.781") fetching...
[relaton-itu] WARNING: no match found online for ITU-T H.781. The code must be exactly like it is on the standards website.
``

**Comment by ronaldtse** [comment]
```sh
[relaton-itu] WARNING: no match found online for ITU-T H.760. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T H.760 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
[relaton-itu] ("ITU-T X.1255") fetching...
[relaton-itu] WARNING: no match found online for ITU-T X.1255. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T X.1255 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
```

**Comment by ronaldtse** [comment]
```
[info]: Compiling /Users/mulgogi/src/mn/mn-samples-itu/sources/T-REC-J.1108-201901-P.MSW-E-draft.adoc ...
[relaton-itu] ("ITU-T J.210") fetching...
[relaton-itu] WARNING: no match found online for ITU-T J.210. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T J.210 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
[relaton-itu] ("ITU-T J.222.1") fetching...
[relaton-itu] WARNING: no match found online for ITU-T J.222.1. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T J.222.1 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
[relaton-itu] ("ITU-T J.222.2") fetching...
[relaton-itu] WARNING: no match found online for ITU-T J.222.2. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T J.222.2 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
[relaton-itu] ("ITU-T J.1106") fetching...
[relaton-itu] WARNING: no match found online for ITU-T J.1106. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T J.1106 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
[relaton-itu] ("ITU-T J.1107") fetching...
[relaton-itu] WARNING: no match found online for ITU-T J.1107. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T J.1107 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
[relaton-itu] ("ITU-T X.743") fetching...
[relaton-itu] WARNING: no match found online for ITU-T X.743. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T X.743 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
```

**Comment by ronaldtse** [comment]
```
[relaton-itu] ("ITU-T G.1022") fetching...
[relaton-itu] WARNING: no match found online for ITU-T G.1022. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T G.1022 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
[relaton-itu] ("ITU-T H.264") fetching...
[relaton-itu] WARNING: no match found online for ITU-T H.264. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T H.264 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
[relaton-itu] ("ITU-T P.800.1") fetching...
[relaton-itu] WARNING: no match found online for ITU-T P.800.1. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T P.800.1 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
[relaton-itu] ("ITU-T P.910") fetching...
[relaton-itu] WARNING: no match found online for ITU-T P.910. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T P.910 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
[relaton-itu] ("ITU-T P.911") fetching...
[relaton-itu] WARNING: no match found online for ITU-T P.911. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T P.911 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
[relaton-itu] ("ITU-T P.1201.1") fetching...
[relaton-itu] WARNING: no match found online for ITU-T P.1201.1. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T P.1201.1 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
[relaton-itu] ("ITU-T P.1201.2") fetching...
[relaton-itu] WARNING: no match found online for ITU-T P.1201.2. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T P.1201.2 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
[relaton-itu] ("ITU-T P.1202") fetching...
[relaton-itu] WARNING: no match found online for ITU-T P.1202. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T P.1202 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
[relaton-itu] ("ITU-T P.1202.1") fetching...
[relaton-itu] WARNING: no match found online for ITU-T P.1202.1. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T P.1202.1 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
[relaton-itu] ("ITU-T P.1203.1") fetching...
[relaton-itu] WARNING: no match found online for ITU-T P.1203.1. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T P.1203.1 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
[relaton-itu] ("ITU-T P.1203.2") fetching...
[relaton-itu] WARNING: no match found online for ITU-T P.1203.2. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T P.1203.2 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
[relaton-itu] ("ITU-T P.1203.3") fetching...
[relaton-itu] WARNING: no match found online for ITU-T P.1203.3. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T P.1203.3 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
[relaton-itu] ("ITU-T P.1401") fetching...
[relaton-itu] WARNING: no match found online for ITU-T P.1401. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T P.1401 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
```

There seems to be some systematic problem.

@andrew2net can you go through the references from mn-samples-itu repo? Thanks.

**Comment by andrew2net** [comment]
@ronaldtse I cant't reproduce it. Did you try to update all gem to last version?
```
[relaton-itu] ("ITU-T G.650.2") fetching...
[relaton-itu] ("ITU-T G.650.2") found ITU-T G.650.2
[relaton-itu] ("ITU-T G.651.1") fetching...
[relaton-itu] ("ITU-T G.651.1") found ITU-T G.651.1
[relaton-itu] ("ITU-T G.652") fetching...
[relaton-itu] ("ITU-T G.652") found ITU-T G.652
[relaton-itu] ("ITU-T G.653") fetching...
[relaton-itu] ("ITU-T G.653") found ITU-T G.653
[relaton-itu] ("ITU-T G.654") fetching...
[relaton-itu] ("ITU-T G.654") found ITU-T G.654
[relaton-itu] ("ITU-T G.655") fetching...
[relaton-itu] ("ITU-T G.655") found ITU-T G.655
[relaton-itu] ("ITU-T G.656") fetching...
[relaton-itu] ("ITU-T G.656") found ITU-T G.656
[relaton-itu] ("ITU-T G.657") fetching...
[relaton-itu] ("ITU-T G.657") found ITU-T G.657
[relaton-itu] ("ITU-T G.671") fetching...
[relaton-itu] ("ITU-T G.671") found ITU-T G.671
[relaton-iec] ("IEC 60793-1-1") fetching...
[relaton-iec] ("IEC 60793-1-1") found IEC 60793-1-1:2017 RLV
[relaton-iec] ("IEC 60793-1-30") fetching...
[relaton-iec] ("IEC 60793-1-30") found IEC 60793-1-30:2010
[relaton-iec] ("IEC 60793-1-42") fetching...
[relaton-iec] ("IEC 60793-1-42") found IEC 60793-1-42:2013
[relaton-iec] ("IEC 60793-1-44") fetching...
[relaton-iec] ("IEC 60793-1-44") found IEC 60793-1-44:2011
[relaton-iec] ("IEC 60793-1-45") fetching...
[relaton-iec] ("IEC 60793-1-45") found IEC 60793-1-45:2017 RLV
[relaton-iec] ("IEC 60793-1-46") fetching...
[relaton-iec] ("IEC 60793-1-46") found IEC 60793-1-46:2001
[relaton-iec] ("IEC 60793-1-47") fetching...
[relaton-iec] ("IEC 60793-1-47") found IEC 60793-1-47:2017 RLV
[relaton-iec] ("IEC 61745") fetching...
[relaton-iec] ("IEC 61745") found IEC 61745:2017
[relaton-iec] ("IEC 61746-1") fetching...
[relaton-iec] ("IEC 61746-1") found IEC 61746-1:2009
[relaton-itu] ("ITU-T G.Suppl.47") fetching...
[relaton-itu] WARNING: no match found online for ITU-T G.Suppl.47. The code must be exactly like it is on the standards website.
[relaton-itu] If you wanted to cite all document parts for the reference, use "ITU-T G.Suppl.47 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).
```
The reference `ITU-T G.Suppl.47` should be `ITU-T G Suppl.47`.

**Comment by ronaldtse** [comment]
Got it let me try again.

> The reference ITU-T G.Suppl.47 should be ITU-T G Suppl.47.

Could Relaton issue a warning for ITU `Suppl` and ITU `Annex` on the correct format? Thanks!

**Comment by ronaldtse** [comment]
Yes it works. I've fixed these references in metanorma/mn-samples-itu@8d53838.
